### PR TITLE
[kotlin/es] Translated Kotlin to spanish

### PR DESCRIPTION
**Pull request by AlburIvan**
- [x] PR touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: es-es/kotlin-es.html.markdown [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+object ObjectExample {
+    fun hello() : String {
+        return "hello"
+    }
+}
+```
+
+### Lectura Adicional
+
+* [Kotlin tutorials](https://kotlinlang.org/docs/tutorials/)
```
Comment (vendethiel) [inline es-es/kotlin-es.html.markdown]: please mark those with a " (EN)" tag (or use spanish links)

Comment (AlburIvan) [inline es-es/kotlin-es.html.markdown]: Tags added, something else?

Comment (vendethiel) [inline es-es/kotlin-es.html.markdown]: Looks good. Can I merge or do you still need to address a few things!

Comment (AlburIvan) [inline es-es/kotlin-es.html.markdown]: Please do, everything is in order from my side  :)

**Comment by vendethiel** [comment]
Thank you! :star:

**Pull request merged**